crates/web5/src/credentials/presentation_definition.rs

```diff
@@ -237,7 +237,7 @@ impl JsonSchemaBuilder {
 mod tests {
     use std::collections::HashSet;
 
-    use test_helpers::TestVectorFile;
+    use crate::test_helpers::TestVectorFile;
```

Comment:
@diehuxx thoughts on

- `crates/`
    - `test_helpers`
    - `web5`

?

Reply:
Or, thoughts on simply 

- `core/`
    - `src/`
    - `Cargo.toml`
    - etc.

Meaning, just lean hard into only a single crate? 

Any of the above three are fine because the compiler will handle performance matters, so it's purely cosmetic, just ideating